app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -319,6 +319,13 @@
     miq_tabs_show_hide("#ssh_keypair_tab", true);
     miq_tabs_init('#auth_tabs');
     $('#auth_tabs').show();
+%div{"ng-if" => "#{ng_model}.emstype == 'vmware_cloud'"}
```

Comment:
@gberginc wondering if there is any way to make this block and few blocks above this block generic, there is a lot of repetition of JS calls here.